tests/attention/test_mla_lse_base.py

```
@@ -0,0 +1,545 @@
+def _mla_reference_lse_natural(
+    query: torch.Tensor,
+    kv_cache: torch.Tensor,
+    block_tables: torch.Tensor,
+    seq_lens: torch.Tensor,
+) -> torch.Tensor:
+    """Natural-log LSE, [batch * q_len, num_heads], accumulated in fp32.
+
+    MLA post-absorption is a single 576-wide dot product against the compressed KV
+    row, so the reference needs no nope/rope split.
+    """
+    batch_size, q_len, num_heads, _ = query.shape
```

Ruff (0.16.1)

[warning] Unpacked variable `q_len` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)